src/processor/processor.cpp

```diff
@@ -75,6 +75,7 @@ void QueryProcessor::decomposePlanIntoTasks(
     case PhysicalOperatorType::SET_NODE_PROPERTY:
     case PhysicalOperatorType::SET_REL_PROPERTY:
     case PhysicalOperatorType::CREATE_NODE:
+    case PhysicalOperatorType::COPY_TO:
```

Comment:
move this line under all `updates` and before `STANDALONE_CALL`. So that we have operators of the same type written together